msm/skill_repo.py

```diff
@@ -111,8 +112,9 @@ def load_skills_data(branch, path):
 
 
 class SkillRepo(object):
-    def __init__(self, path=None, url=None, branch=None):
-        self.path = path or "/opt/mycroft/.skills-repo"
+    def __init__(self, url=None, branch=None):
+        self.path = join(BaseDirectory.save_data_path('mycroft'),
+                         '.skills-repo')
```

Comment:
Actually I'm wondering, is there a need to keep this folder hidden? Why was it in the first place?

Reply:
I believe it was hidden to mark it as something that shouldn't be edited, since it was in the opt/mycroft folder. Since this is in a "save data" it doesn't need to be hidden in my opinion. The save data path should be enough of an indication that it shouldn't be tampered with :)